modules/node_modules/@ncigdc/components/Case.js

```diff
@@ -281,13 +282,14 @@ const Case = ({
         <LocationSubscriber>{(ctx: {| pathname: string, query: TRawQuery |}) => {
           const { filters } = ctx.query || {};
           const currentFilters = parseFilterParam(filters, { op: 'and', content: [] });
+          const componentFilters = _.mergeWith(currentFilters, setFilter({ field: 'cases.case_id', value: p.case_id }), (a, b) => (Array.isArray(a) ? a.concat(b) : undefined));
```

Comment:
Might be good to using the methods in the filters utils. something like this https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/components/Oncogrid/getQueries.js#L19